src/lib/utils/noop.ts

```diff
@@ -0,0 +1,3 @@
+function noop() {}
+
+export default noop;
```

Comment:
typing wise, do you knwo why they have `noop(...args: any[]): void;` in lodash and if there is any benefit to it?
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/06d8677cde28467e830bf77bc2cbfdaca37740aa/types/lodash/common/util.d.ts#L885

Reply:
this means that you can call a noop without issues; I think we might not allow our noop to be called

Reply:
I guess it's to be more flexible. I'll update that.

Reply:
Hmm I didn't see your comment @Haroenv. I updated the signature.